packages/react-components/src/layout/src/Inline.tsx

```diff
@@ -52,29 +57,29 @@ export function InnerInline({
     wrap,
     forwardedRef,
     ...rest
-}) {
+}: InnerInlineProps): ReactElement {
     const alignProps = useFlexAlignment("horizontal", align, verticalAlign);
 
+    const extraFlexProps: Partial<FlexProps> = {
```

Comment:
we can't type inline elements, so we extract it and type it before the render function

Reply:
The alternative for the inline is 

```
{
    gap: gap !== 0 ? gap : undefined,
    wrap: !isNil(wrap) ? "wrap" : undefined,
    ref: forwardedRef
} as Partial<FlexProps>
``` 

However, since it's a type casting and not a typing, there is no validation that the value passed are actually valid